File: CS/WebSite/NullTextUserControl.ascx.cs
using System;
using System.Collections.Generic;
using System.Linq;


public partial class NullTextUserControl : System.Web.UI.UserControl {
    private string themePrefix;

    public string ThemePrefix {
        get { return themePrefix; }
        set { themePrefix = value; }
    }
}

File: VB/WebSite/NullTextUserControl.ascx.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq


Partial Public Class NullTextUserControl
	Inherits System.Web.UI.UserControl
	Private themePrefix_Renamed As String

	Public Property ThemePrefix() As String
		Get
			Return themePrefix_Renamed
		End Get
		Set(ByVal value As String)
			themePrefix_Renamed = value
		End Set
	End Property
End Class